CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ This document outlines major changes between releases.
 
 ### Changed
 - Go 1.22+ is required to build now (#927)
+- stream_timeout default timeout (#973)
```

Comment:
It's be nice to mention new timeout value.

Reply:
It makes sense, updated